native/src/routes/Pois.tsx

```diff
@@ -177,11 +128,8 @@ const Pois = ({ pois: allPois, language, cityModel, route, navigation, refresh }
     </>
   )
 
-  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-  const currentBottomSheetHeight = snapPoints[sheetSnapPointIndex]!
-
   return (
-    <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
+    <View style={{ flex: 1 }}>
```

Comment:
❓ Does the styling still need to be inline here?

Reply:
I don't think so, good point :+1: